pipeline.py

```diff
@@ -392,12 +392,6 @@ def ilab_pipeline(
     run_mt_bench_task.set_accelerator_limit(1)
     run_mt_bench_task.set_caching_options(False)
     run_mt_bench_task.after(training_phase_2)
-    use_config_map_as_env(
```

Comment:
Should this have been removed to maintain backwards compatibility?

Reply:
Not sure I'm following, backwards compatibility for what?

Reply:
The code under `if judge_secret_name is None:` relies on an environment variable of `JUDGE_ENDPOINT` and `JUDGE_NAME` does it not?

Reply:
oh I see what you mean, the backwards compatibility comment I left in the PR description refers to only standalone.py, which will utilize the same component code for sdg/mt_eval/final_eval but will mount the configmaps/secrets as env vars, for example for sdg this is done [here](https://github.com/opendatahub-io/ilab-on-ocp/blob/d9678da2a8939c18d0f2a363789559cbd3ca3534/standalone/standalone.py#L1291), we want to maintain compatibility with standalone.py

from the pipeline's perspective, you provide a secret name, and we will use it, how we use it is an implementation detail